scripts/rpc-maas-tool.py

```diff
@@ -671,8 +671,9 @@ def _get_failed_checks(self, checks=None):
                 try:
                     result = self.rpcm.conn.test_existing_check(check)
                 except rackspace.RackspaceMonitoringValidationError as e:
-                    validation_error = (" Validation Error: %(s):"
-                                        % {'e': e.message})
+                    validation_error = " Validation Error: {s}:".format(
+                        s=e.message
+                    )
                     break
```

Comment:
Fixing the key error means this `break` line gets executed, which in turn means the for-loop ends and failed_checks is never updated. At least with the key error the script will fail, this patch is going to cause _get_failed_checks to misreport the status of the checks.